[2023-06-27T14:48:03.676+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe permission to 509
[2023-06-27T14:48:03.677+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe'
[2023-06-27T14:48:03.707+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe permission to 509
[2023-06-27T14:48:03.708+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe'
[2023-06-27T14:48:03.852+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.create_pipe manual__2023-06-27T14:47:24.611246+00:00 [queued]>
[2023-06-27T14:48:03.862+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.create_pipe manual__2023-06-27T14:47:24.611246+00:00 [queued]>
[2023-06-27T14:48:03.863+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-27T14:48:03.876+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_pipe> on 2023-06-27 14:47:24.611246+00:00
[2023-06-27T14:48:03.881+0000] {standard_task_runner.py:57} INFO - Started process 2856 to run task
[2023-06-27T14:48:03.885+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Setup', 'create_pipe', 'manual__2023-06-27T14:47:24.611246+00:00', '--job-id', '308', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpg5xc3v2_']
[2023-06-27T14:48:03.887+0000] {standard_task_runner.py:85} INFO - Job 308: Subtask create_pipe
[2023-06-27T14:48:03.930+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe permission to 509
[2023-06-27T14:48:03.930+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=create_pipe'
[2023-06-27T14:48:03.933+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Setup.create_pipe manual__2023-06-27T14:47:24.611246+00:00 [running]> on host eb22b15bae36
[2023-06-27T14:48:04.052+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Setup' AIRFLOW_CTX_TASK_ID='create_pipe' AIRFLOW_CTX_EXECUTION_DATE='2023-06-27T14:47:24.611246+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-27T14:47:24.611246+00:00'
[2023-06-27T14:48:04.054+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-27T14:48:04.055+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run-operation create_pipe --args \'{                            "pipe_name": "get_new_files",                             "table_name": "ed_pipeline_logs",                             "dry_run": false}\'']
[2023-06-27T14:48:04.066+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T14:48:07.156+0000] {subprocess.py:93} INFO - [0m14:48:07  Running with dbt=1.5.1
[2023-06-27T14:48:14.686+0000] {subprocess.py:93} INFO - [0m14:48:14  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-27T14:48:14.687+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2023-06-27T14:48:14.688+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day_lu
[2023-06-27T14:48:14.746+0000] {subprocess.py:93} INFO - [0m14:48:14  Found 3 models, 0 tests, 0 snapshots, 0 analyses, 456 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-27T14:48:16.718+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-27T14:48:16.747+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Setup, task_id=create_pipe, execution_date=20230627T144724, start_date=20230627T144803, end_date=20230627T144816
[2023-06-27T14:48:16.813+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-27T14:48:16.836+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
